src/OpenTelemetry.Instrumentation.Runtime/README.md

```diff
@@ -28,16 +28,16 @@ Runtime instrumentation should be enabled at application startup using the
 ```csharp
 using var meterProvider = Sdk.CreateMeterProviderBuilder()
     .AddRuntimeInstrumentation()
-    .AddPrometheusExporter()
+    .AddPrometheusHttpListener()
     .Build();
 ```
 
 Refer to [Program.cs](../../examples/runtime-instrumentation/Program.cs) for a
 complete demo.
 
-Additionally, this examples sets up the OpenTelemetry Prometheus exporter, which
-requires adding the package
-[`OpenTelemetry.Exporter.Prometheus`](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Exporter.Prometheus.HttpListener/README.md)
+Additionally, this examples sets up the OpenTelemetry Prometheus Exporter
```

Comment:
```suggestion
Additionally, the above example snippet sets up the OpenTelemetry Prometheus Exporter
```

Reply:
or skip showing prometheus in this readme completely, and mention that "A full demo app is shown (link), which also adds prometheus exporter.." .

Not a PR blocker, we can adjust these separately as well.